frontend/components/ui/textarea.tsx

```
@@ -4,14 +4,30 @@ import { cn } from "@/lib/utils";
 const Textarea = React.forwardRef<
   HTMLTextAreaElement,
   React.TextareaHTMLAttributes<HTMLTextAreaElement>
->(({ className, ...props }, ref) => {
+>(({ className, onChange, ...props }, ref) => {
+  const [error, setError] = React.useState<string | null>(null);
```

ESLint

[error] 'error' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)